### PR TITLE
Added valuesets for coverage

### PR DESCRIPTION
**Pull request by Gokulsignal**
@enjoysparkling , Hi Matt, Kindly review the pull request.

### CODE REVIEW

File: input/vocabulary/valueset-coverage-class-code.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-coverage-class-code.json:1]: Changes required:
```
Message  : Type checking the data: Encountered unknown element 'definition' at location 'ValueSet.expansion[0].contains[0].definition[0]' while parsing
FilePath : input\vocabulary\valueset-coverage-class-code.json
Id       : 1003
Severity : Error
Context  : ValueSet.expansion[0].contains[0]
```

Also include SYSTEM in the expansion.

File: input/vocabulary/valueset-coverage-copay-type-code.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-coverage-copay-type-code.json:1]: Changes required:
```
Message  : Instance failed constraint vsd-10 "Must have a system if a code is present"
FilePath : input\vocabulary\valueset-coverage-copay-type-code.json
Id       : vsd-10
Severity : Error
Context  : ValueSet.expansion[0].contains[0]
```

File: input/vocabulary/valueset-coverage-financial-exception.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-coverage-financial-exception.json:1]: Changes required:
```
Message  : Instance failed constraint vsd-10 "Must have a system if a code is present"
FilePath : input\vocabulary\valueset-coverage-financial-exception.json
Id       : vsd-10
Severity : Error
Context  : ValueSet.expansion[0].contains[0]
```

File: input/vocabulary/valueset-subscriber-relationship.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-subscriber-relationship.json:1]: Changes required:
```
Message  : Instance failed constraint vsd-10 "Must have a system if a code is present"
FilePath : input\vocabulary\valueset-subscriber-relationship.json
Id       : vsd-10
Severity : Error
Context  : ValueSet.expansion[0].contains[0]
```

**Review by enjoysparkling — CHANGES_REQUESTED** [review]

**Comment by Gokulsignal** [comment]
@enjoysparkling , Hi Matt, the above changes are made for the Coverage Value sets. Please review.

**Review by enjoysparkling — APPROVED** [review]

**Pull request merged**